js/render/draw_extrusion.js

```diff
@@ -163,7 +163,7 @@ function drawExtrusion(painter, source, layer, coord) {
     );
 
     if (image) {
-        setPattern(image, tile, coord, painter, program);
+        setPattern(image, tile, coord, painter, program, true);
```

Comment:
Can we tweak the argument signature to avoid a [boolean trap](https://ariya.io/2011/08/hall-of-api-shame-boolean-trap) here? Perhaps 

```js
setPattern(image, tile, coord, painter, program, {includeHeightFactor: true})
```